### PR TITLE
Prevent users below a recaptchaRating threshold to submit posts and c…

### PR DESCRIPTION
**Pull request by Discordius**
…omments and hide them from the sunshine sidebar

### CODE REVIEW

File: packages/lesswrong/components/common/RecaptchaWarning.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  children: React.ReactNode
+}) => {
+  if (!currentUser?.spamRiskScore || (currentUser.spamRiskScore > spamRiskScoreThreshold)) {
+    return <> { children } </>
+  }
+  // ea-forum look here: You will want to change where this links to
+  return <div className={classes.warningText}>
+    You've been flagged by our spam detection system. Please message an admin via 
+    Intercom (the chat bubble in the bottom right corner) or send a private message to admin 
+    <Link className={classes.link} to="/users/habryka"> habryka</Link> to activate posting- and commenting-privileges on your account.
```
Comment (Raemon) [inline packages/lesswrong/components/common/RecaptchaWarning.tsx:33]: should this just be a setting field?

**Comment by Raemon** [comment]
This didn't seem to actually prevent the low-recaptcha users from showing up in sunshine sidebar:

image

File: packages/lesswrong/components/common/RecaptchaWarning.tsx (hunk 1 above)
Comment (Discordius) [reply packages/lesswrong/components/common/RecaptchaWarning.tsx:33]: Yeah, but the settings PR isn't merged and it should be a database settings field. As soon as we merge that PR I can change things here.

**Comment by Discordius** [comment]
Works for me. Are you sure you were actually on the correct branch?

**Comment by Raemon** [comment]
Ah, I accidentally created a new branch instead of downloading yours. Now it looks like it's working right.

**Pull request merged**